lib/v2/huggingface/blog-zh.js

```diff
@@ -0,0 +1,34 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { parseDate } = require('@/utils/parse-date');
+
+module.exports = async (ctx) => {
+    const { body: response } = await got('https://huggingface.co/blog/zh');
+    const $ = cheerio.load(response);
+
+    /** @type {Array<{blog: {local: string, title: string, author: string, thumbnail: string, date: string, tags: Array<string>}, blogUrl: string, lang: 'zh', link: string}>} */
+    const papers = [];
+
+    $('div[data-target="BlogThumbnail"]').each(function () {
+        const props = $(this).data('props');
+        const link = $(this).find('a').attr('href');
+        papers.push({
+            ...props,
+            link,
+        });
+    });
```

Comment:
```suggestion
    const papers = $('div[data-target="BlogThumbnail"]')
        .toArray()
        .map((item) => {
            const props = $(item).data('props');
            const link = $(item).find('a').attr('href');
            return {
                ...props,
                link,
            };
        });
```
Please use the [faster](https://www.measurethat.net/Benchmarks/Show/11532/0/push-vs-map) `.map()` instead.